src/BattleTest.java

```diff
@@ -0,0 +1,117 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.function.Executable;
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BattleTest {
+    private PrintStream printer;
+
+    @BeforeEach
+    public void printerSetup(){
+        printer = new PrintStream(new OutputStream() {
+                @Override
+                public void	write(int b) {
+                    // noop
+                }
+            });
```

Comment:
Formázásra figyelj.

Reply:
![pic](https://user-images.githubusercontent.com/60850100/114908830-c701f800-9e1c-11eb-84ee-9f0de284a6c9.JPG)

Hát ez érdekes mert nekem így van. De előzőleg is már. Direkt figyeltem, hogy most jó legyen és még mindig rossz.    -   ?

Reply:
```java
    public void printerSetup() {
        printer = new PrintStream(new OutputStream() {
            @Override
            public void	write(int b) {
                // noop
            }
        });
    }
```
De majd ha ez a PR be lesz húzva és megcsinálom a gradle-t, akkor majd CI-ben automatizálva lesz ez a folyamat is.